mlrun/__main__.py

```diff
@@ -1389,5 +1410,21 @@ def func_url_to_runtime(func_url, ensure_project: bool = False):
     return runtime
 
 
+def add_notification2proj(notification, proj):
```

Comment:
I know you took inspiration from other method names here, but try to be explicit and adhere to naming conventions :) 
```suggestion
def add_notification_to_project(notification, project):
```